src/rb3/unk_8000D980.cpp

```diff
@@ -0,0 +1,13 @@
+#include "string.hpp"
+#include "textfile.hpp"
+#include "textstream.hpp"
+#include "unknown.hpp"
+
+// fn_8000DB9C
+const char* String::GetText() const {
+    return text;
+}
```

Comment:
i suspect inlining here. leave a comment noting this may be an inline from a header file